### ISSUE TITLE
Obtain similar performance as Kaggle's notebook(s)

### ISSUE THREAD

**Issue by Galeos93** [Galeos93/disaster_tweets#7]
Try to obtain same performance as https://www.kaggle.com/pinkaxe/pytorch-with-embeddingbag-layer, which is very similar to your approach. I don't obtain such high accuracy on validation set.

**Comment by Galeos93** [comment]
After code checking, I realized the vocabulary of train and val sets were totally disjunct! Big mistake.

**Comment by Galeos93** [comment]
I have to see how to make a torchtext vocabulary to deal with out of vocab words.

**Comment by Galeos93** [comment]
Solved
